packages/docs/components/modal.md

```diff
@@ -24,50 +24,6 @@ links:
 
 <Anatomy img="/images/anatomy-modal.svg" />
 
-<Description>
-
-### Curtain
-
-Covers the main content to provide focus to the information served in the dialog.
-
-### Dialog
-
-The Modal itself; interactions are limited to this space.
-
-### Title
-
-Reinforces context from the in-page trigger while providing an overview of the modal.
-
-### Content area
-
-This is where the content of the modal dialog is held. Keep it simple; the need for Tabs or navigation is a warning sign.
-
-### Action bar
-
-Contains the primary action of the modal. All modals must have a primary CTA, and support secondary CTAs as well. Action labels should clarify the task users are being asked to complete.
-
-</Description>
-
-## Behavior
```

Comment:
I thought we were keeping behavior, just cutting Anatomy notes.

Reply:
Youre right. Will fix.